tcms/testruns/templates/testruns/mutable.html

```diff
@@ -63,7 +63,7 @@
         <div class="form-group">
             <label class="col-lg-12" for="id_notes">{% trans "Notes" %}</label>
             <div class="col-lg-12">
-                <textarea class="form-control" id="id_notes" name="notes">{{ form.notes.value }}</textarea>
+                <textarea class="form-control" id="id_notes" name="notes">{{ object.notes }}</textarea>
```

Comment:
Why change and remove `.value` ?

Reply:
Just preferred to use the model iteself.

Reply:
You can't just prefer. There must be a reason for the change and the change must be actually working.